eureka2-integration/src/test/java/com/netflix/eureka2/integration/server/batching/ReadServerNotificationBatchingTest.java

```diff
@@ -58,7 +60,10 @@ public void testColdReadCacheDataBatching() throws Exception {
         eurekaDeploymentClients.fillUpRegistry(REGISTRY_INITIAL_SIZE, SampleInstanceInfo.WebServer.build());
 
         // Bootstrap read server and connect Eureka client immediately
-        eurekaDeploymentResource.getEurekaDeployment().getReadCluster().scaleUpByOne();
+        int serverId = eurekaDeploymentResource.getEurekaDeployment().getReadCluster().scaleUpByOne();
+        EmbeddedReadServer newServer = eurekaDeploymentResource.getEurekaDeployment().getReadCluster().getServer(serverId);
+        assertThat(newServer.waitForUpStatus(5, TimeUnit.SECONDS), is(true));
```

Comment:
each server exposes status notification stream via web socket interface. We can also provide direct access to status aggregator component via EurekaServer API, and make it accessible via EurekaServerRunner.